src/v2/BaseLoginRouter.js

```diff
@@ -195,7 +190,11 @@ export default Router.extend({
           this.render(Controller, options);
         })
         .catch(errorResp => {
+          // clear session storage regardless there exists
+          // a state handle due to error.
+          sessionStorageHelper.removeStateHandle();
```

Comment:
this introspect failure could be because of invalid existing stateToken (previously saved).. Hence, In the catch we can again call (2nd attempt) `startLoginFlow` with new stateToken that widget just received from server..

Reply:
This has been handled at `startLoginFlow`